components/datetime/src/format/neo.rs

```diff
@@ -545,6 +547,8 @@ pub(crate) struct RawDateTimeNames<R: DateTimeNamesMarker> {
         >,
     zone_essentials:
         <R::ZoneEssentials as DateTimeNamesHolderTrait<tz::EssentialsV1Marker>>::Container<()>,
+    locations_root:
+        <R::ZoneLocations as DateTimeNamesHolderTrait<tz::LocationsV1Marker>>::Container<()>,
```

Comment:
Thought: This makes `DateTimeFormatter<YMDHMSZ>` a little bit bigger in stack size.